Trying:
    say_my_name = __import__('3-say_my_name').say_my_name
Expecting nothing
ok
Trying:
    say_my_name("Pouquerou", "Baptiste")
Expecting:
    My name is Pouquerou Baptiste
ok
Trying:
    say_my_name("Pouquerou")
Expecting:
    My name is Pouquerou 
ok
Trying:
    say_my_name("", "")
Expecting:
    My name is  
ok
Trying:
    say_my_name(12, "Baptiste")
Expecting:
    Traceback (most recent call last):
    TypeError: first_name must be a string
ok
Trying:
    say_my_name("Pouquerou", 123)
Expecting:
    Traceback (most recent call last):
    TypeError: last_name must be a string
ok
Trying:
    say_my_name(True, True)
Expecting:
    Traceback (most recent call last):
    TypeError: first_name must be a string
ok
Trying:
    say_my_name(None, "Baptiste")
Expecting:
    Traceback (most recent call last):
    TypeError: first_name must be a string
ok
Trying:
    say_my_name()
Expecting nothing
**********************************************************************
File "./tests/3-say_my_name.txt", line 52, in 3-say_my_name.txt
Failed example:
    say_my_name()
Exception raised:
    Traceback (most recent call last):
      File "/usr/lib/python3.10/doctest.py", line 1350, in __run
        exec(compile(example.source, filename, "single",
      File "<doctest 3-say_my_name.txt[8]>", line 1, in <module>
        say_my_name()
    TypeError: say_my_name() missing 1 required positional argument: 'first_name'
**********************************************************************
1 items had failures:
   1 of   9 in 3-say_my_name.txt
9 tests in 1 items.
8 passed and 1 failed.
***Test Failed*** 1 failures.
